agent/config/config.go

```diff
@@ -563,7 +577,11 @@ func (cfg *Config) validateAndOverrideBounds() error {
 	}
 
 	if cfg.DockerStopTimeout < minimumDockerStopTimeout {
-		return fmt.Errorf("Invalid negative DockerStopTimeout: %v", cfg.DockerStopTimeout.String())
+		return fmt.Errorf("invalid DockerStopTimeout: %v", cfg.DockerStopTimeout.String())
```

Comment:
can you make this more human readable? `"config: invalid value for stop timeout..."`

Reply:
OK, sure